pkgs/tools/networking/nttcp/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchurl, ... }:
```

Comment:
@fpletz please review more carefully, you missed the ... here.

Reply:
Apologies. I will make sure to remove it from all future commits